docs/guides/main.rst

```diff
@@ -139,6 +146,11 @@ place to start::
     directory: ~/music
     library: ~/data/musiclibrary.db
 
+For Windows user, this would look like::
+
+    directory: "D:\\My Music Files"
+    library: "D:\\My Music Files\\Musiclibrary.db"
```

Comment:
IMHO, unixy paths are not natural to dominantly Windows users. And translating Windows paths to Unix format is even less so. When I read the guide, that was the first question that popped up in my mind. 

May I suggest we add an example that shows how paths with spaces in them are used? Or if they can be used as-is, worth explicitly clarifying that in the text?